src/components/containers/ChartContainer.js

```diff
@@ -22,7 +22,31 @@ export default class extends React.Component {
 
         // Show outliers toggle constants.
         this.outliersThreshold = 10;
-        this.outliersSmallestProportion = 0.01;
+        this.outliersSmallestProportion = 0.0001;
```

Comment:
Out of curiosity, why has this value changed?

Reply:
We were comparing the values while multiplied by 100 before. Now we're comparing them as raw from the server response.